### PR TITLE
Added Post Incident Practices module

### PR DESCRIPTION
**Pull request by tom-halpin**
## Proposed changes

Added Post Incident Practices module

## Types of changes

What types of changes does your code introduce to Online DevOps Dojo?
_Put an `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Workflow change
- [ ] Documentation update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [X] I have read the [CONTRIBUTING](https://github.com/dxc-technology/online-devops-dojo/blob/master/CONTRIBUTING.md) doc.
- [X] Lint passes locally with my changes.
- [X] I have added necessary documentation (if appropriate).
- [X] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by tom-halpin** [comment]
@cpswan @ojacques @pberthonneau open sourcing Post Incident Practices module by way of a small thank you to the community given the very positive reaction to the Dojo since we launched it.

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.5.2] - 2020-12-15
```
Comment (adamdmharvey) [inline CHANGELOG.md:8]: If aligning to SemVer, would suggest adding a new module would be a minor change (new feature, backward compatiable, non-breaking), and so would suggest bump to 1.6.0.  Third digit normally reserved for bug fixes.

File: online-devops-dojo/welcome/intro.md [hunk 2]
```diff
@@ -6,14 +6,15 @@ Welcome. I am the Octocat for the **Online DevOps Dojo** Katacoda course
 
 You are about to enter another dimension, a dimension not only of sight and sound but of mind. A journey into a wondrous land of imagination. Next stop, the **Online DevOps Dojo**!
 
-This course is made up of 6 open source modules.
+This course is made up of 7 open source modules.
 
 * Leading Change
```
Comment (adamdmharvey) [inline online-devops-dojo/welcome/intro.md:11]: Just cross referencing with https://github.com/dxc-technology/about-devops-dojo/pull/64, the announcement suggests 7 modules with this post incident practices being the newest No 8.  This list says 6 current + 1 new or 7 total.  Looks like the difference is the announcement says Welcome and Setup is the first module.

Just suggest to align the two?

**Review by adamdmharvey — CHANGES_REQUESTED** [review]
Looks great!  Two minor points from my side but I can see this being super valuable in the community!

File: CHANGELOG.md (hunk 1 above)
Comment (tom-halpin) [reply CHANGELOG.md]: @adamdmharvey good point, change made

File: online-devops-dojo/welcome/intro.md (hunk 2 above)
Comment (tom-halpin) [reply online-devops-dojo/welcome/intro.md:11]: @adamdmharvey again good point, change made.

**Pull request merged**